e2e/tests/009.sh

```diff
@@ -87,14 +87,14 @@ while [[ $retries -gt 0 ]]; do
 
     modified=false
     echo "Pushing update"
-    output=$(kpt alpha rpkg push -n default "$smf_pkg_rev" $ws 2>&1)
+    output=$(kpt alpha rpkg push -n default "$smf_pkg_rev" $ws 2>&1 || rc=$?)
```

Comment:
I don't think that we are using `rc` anywhere in this script
```suggestion
    output=$(kpt alpha rpkg push -n default "$smf_pkg_rev" $ws 2>&1 /dev/null)
```